torch/csrc/jit/export.cpp

```diff
@@ -370,6 +378,182 @@ void validateGraph(const std::shared_ptr<Graph>& graph, onnx::OperatorExportType
   }
 }
 
+// Pretty printing
```

Comment:
Do the standard pretty printing options for protobuf not work here?

For example

https://stackoverflow.com/questions/41651271/is-there-an-example-of-protobuf-with-text-output
and
`message->DebugString()`

If not, that's okay - just wanted to check.